### DISCUSSION TITLE
Tailwind styles not working on Web

### DISCUSSION THREAD

**Discussion by stephanepx** [obytes/react-native-template-obytes#378] [Q&A] [answered]
Hi guys,
Thanks again for this awesome library.

When I launch my app on web : `npx expo start --web --clear`, Tailwind styles are not applied. 
I found this [issue](https://github.com/nativewind/nativewind/issues/470) saying that we need to do something like that : 

```
import { NativeWindStyleSheet } from "nativewind";

NativeWindStyleSheet.setOutput({
  default: "native",
});
```

But `NativeWindStyleSheet` is  not exported anymore in v4...

Do you know how we can fix this ?

Thanks

**Comment by yjose** [comment]
It's just yesterday that the NativeWind team made version 4 the default. We will ensure that we migrate to the latest version as soon as possible.

**Reply by yjose** [reply, marked as answer]
check update [here](https://github.com/obytes/react-native-template-obytes/pull/380)  to migrate to the last nativewind version, this update should fix your issue

**Reply by stephanepx** [reply]
@yjose thanks for your answer. 
I updated all changes like in the Pull Request, but it's still not working on Web
